lib/codecs/Cargo.toml

```diff
@@ -20,6 +20,7 @@ chrono.workspace = true
 rust_decimal = { version = "1.37", default-features = false, features = ["std"] }
 csv-core = { version = "0.1.12", default-features = false }
 derivative.workspace = true
+derive_more = { version = "0.99", optional = true }
```

Comment:
This dependency has since had a 1.x release, probably should bump it and address breaking changes (if any).